sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/ClientContextTests.cs

```diff
@@ -28,13 +28,13 @@ public async Task TestClientContextPolicy()
             
             Console.WriteLine("-----Client 1-----");
             _ = await client1.DefaultSubscription.GetResourceGroups().Construct(LocationData.WestUS2).CreateOrUpdateAsync(Recording.GenerateAssetName("testrg"));
-            Assert.AreEqual(2, dummyPolicy1.numMsgGot);
+            Assert.AreEqual(1, dummyPolicy1.numMsgGot);
```

Comment:
why is this test assert needed?

Reply:
The Assert result here changed because there is one reuqest short after Subscirption change.

Reply:
Fixed by #21069